wander-embedded-sdk/src/components/button/wander-button.component.ts

```diff
@@ -280,11 +282,13 @@ export class WanderButton {
   }
 
   setBalance(balanceInfo: BalanceInfo) {
-    const formattedBalance = new Intl.NumberFormat(undefined, {
-      currency: balanceInfo.currency
-    }).format(balanceInfo.amount);
+    if (balanceInfo.amount === null) {
```

Comment:
If the balance is hidden in the app, it's also hidden here. Also, the real amount is not send, neither in `amount` nor in `formattedBalance`.